modules/photons_control/addon.py

```diff
@@ -110,19 +113,24 @@ async def find_ips(collector, target, reference, artifact, **kwargs):
             if Services.UDP in services:
                 ip = services[Services.UDP].host
                 ips[serial] = ip
-                if cli_output:
-                    print(f"{serial}: {ip}")
+
+    sorted_ips = sorted(ips.items(), key=ip_address)
```

Comment:
we should instead import the ipaddress module and sort by ip address objects so this will work in an ipv6 future

```suggestion
    sorted_ips = sorted(ips.items(), key=lambda item: ipaddress.ip_address(item[1]))
```

(I assume these objects are sortable

Reply:
Turns out your assumption is 100% correct.